ibm/data_source_ibm_scc_posture_v2_group_profile_details.go

```diff
@@ -0,0 +1,362 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	//"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/scc-go-sdk/posturemanagementv2"
+)
+
+func dataSourceIBMSccPostureV2GroupProfileDetails() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMSccPostureV2GroupProfileDetailsRead,
+
+		Schema: map[string]*schema.Schema{
+			"profile_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The profile ID. This can be obtained from the Security and Compliance Center UI by clicking on the profile name. The URL contains the ID.",
+			},
+			"first": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The URL of a page.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"href": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The URL of a page.",
+						},
+					},
+				},
+			},
+			"last": &schema.Schema{
```

Comment:
`first`,`last` and `previous` is not required as this datasource should by default do pagination..
So this info is not required for user